src/components/views/room_settings/AliasSettings.js

```diff
@@ -160,12 +172,42 @@ module.exports = React.createClass({
         }
     },
 
+    addAlias: function(alias) {
+         // add this alias to the domain to aliases dict
+        var domain = alias.replace(/^.*?:/, '');
```

Comment:
I meant it generally, that was just the first one that jumped out, applies to https://github.com/matrix-org/matrix-react-sdk/pull/835/files#diff-26b9eb129e9736bd1599b3c40ff80f57R149 and more